wix-embedded-mysql/src/test/scala/com/wix/mysql/ScriptResolverTest.scala

```diff
@@ -0,0 +1,55 @@
+package com.wix.mysql
+
+import java.io.File
+
+import com.wix.mysql.ScriptResolver.{classPathFiles, classPathFile, ScriptResolutionException}
+import org.specs2.matcher.FileMatchers
+import org.specs2.mutable.SpecWithJUnit
+
+import scala.collection.convert.decorateAsScala._
+
+/**
+ * @author viliusl
+ * @since 06/06/15
+ */
+class ScriptResolverTest extends SpecWithJUnit with FileMatchers {
+
+  //TODO: add additional variations for sorting, etc.
+  "ScriptResolver.classPathFile" should {
+
+    "resolve a single classPath file" in {
+      classPathFile("/db/001_init.sql") must beAFile
+    }
+
+    "resolve a single classPath file without preceding '/'" in {
+      classPathFile("db/001_init.sql")must beAFile
+    }
+
+    "throw a ScriptResolutionException for a non-existent script" in {
+      classPathFile("db/not-exists.sql") must throwA[ScriptResolutionException]
+    }
+  }
+
+  "ScriptResolver.classPathFiles" should {
+
+    "resolve multiple classPath files" in {
+      classPathFiles("/db/*.sql").asScala mustEqual Seq(
```

Comment:
Does not add anything:

```
mustEqual Seq(...)
```

to 

```
must containTheSameElementsAs(Seq(...))
```

and then I have to verify order of scripts, so it would be like

```
must contain(exactly(...)).inOrder
```

Does it look better/cleaner? I don't agree to using matchers just for 'using matchers sake'.